tests/test_event_detection.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+import numpy as np
+from scipy.io import loadmat
+
+
+def test_event_comparison():
+    pass
```

Comment:
WIP? Delete? Or are you planning to add something here

Reply:
I forgot about this. WIP I guess, but maybe I can work on a short test here and then revisit the test suite later.

Reply:
I think I'm to save this for a later PR and just open an issue.